lib/mix/test/mix/cli_test.exs

```diff
@@ -47,9 +47,27 @@ defmodule Mix.CLITest do
       end
       """
 
+      File.write! "lib/hello_quiet.ex", """
```

Comment:
We don't need to create a new task! Just change the task above to use Mix.shell.info and then it should work for both calls below. :)